src/components/Admin/AdminLogin/AdminLogin.tsx

```diff
@@ -0,0 +1,57 @@
+import { Form, FormLayout, TextField, Button, Card } from "@shopify/polaris";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { authAccessToken } from "../../../utils/auth";
+
+export interface IAdminLoginProps {
+  setAccessSessionStateToken: React.Dispatch<
+    React.SetStateAction<string | null>
+  >;
+}
+
+export function AdminLogin({ setAccessSessionStateToken }: IAdminLoginProps) {
+  const { t } = useTranslation();
+  const [
+    shouldShowInvalidAccessToken,
+    setShouldShowInvalidAccessToken,
+  ] = useState(false);
+  const [accessToken, setAccessToken] = useState("");
+
+  const handleSubmit = () => {
+    const bearerToken = authAccessToken(accessToken);
+    if (bearerToken) {
+      window.sessionStorage.setItem("bearerToken", bearerToken);
+      setAccessSessionStateToken(bearerToken);
+    } else {
+      setShouldShowInvalidAccessToken(true);
+    }
+  };
+
+  const invalidAccessToken = shouldShowInvalidAccessToken
+    ? t("invalidaccesstoken")
+    : undefined;
+
+  return (
+    <>
```

Comment:
As an FYI, in case you aren't aware, fragments are only required in order to group sibling elements together - JSX always needs to have a single root element, but `<Card>` could be the root in this case.